cmd/cvc-operator/controller/cvc_controller.go

```diff
@@ -263,6 +275,16 @@ func (c *CVCController) createVolumeOperation(cvc *apis.CStorVolumeClaim) (*apis
 		return nil, err
 	}
 
+	if cvclaim.IsHAVolume(cvc) {
```

Comment:
this being small function and doesn't depend on package related things, cvc.IsHAVolume() looks to be good

Reply:
To define cvc.IsHAVolume() we have to define in apis package i.e `pkg/apis/openebs.io/v1alpha1` which is not good practice. So I will define in same `cmd/cvc-operator/controller/cvc_controller.go
` file